front-end/src/Pages/Client/Client-Product/innerPage/PlusMinus.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  AiOutlinePlusCircle,
+  AiOutlineMinusCircle,
+} from 'react-icons/ai';
+import { shoppingListAction } from '../../../../Redux/action/shoppingListAction';
+
+const PlusMinus = ({ index, id, name, photo, price, sellingQuantity, setSellingQuantity }) => {
+  const [disable, setDisable] = useState(false);
+  const dispatch = useDispatch();
+  const shoppingList = useSelector(state => state.shoppingListReducer.data);
+
+  const changeReduxShoppingList = (operator) => {
+    if (shoppingList.length > 0) {
```

Comment:
Acredito que isso pode evitar bugs, caso seja possível que shoppingList não seja um array. De qualquer forma boa por usar o length ```suggestion
    if (shoppingList && shoppingList.length > 0) {
```

Reply:
Feito!